processor/src/main/java/org/mapstruct/ap/internal/model/AbstractBaseBuilder.java

```diff
@@ -50,7 +50,27 @@ public B method(Method sourceMethod) {
         return myself;
     }
 
-    boolean isDisableSubMappingMethodsGeneration() {
+    /**
+     * Checks if MapStruct is allowed to generate an automatic sub-mapping between {@code sourceType} and @{code
```

Comment:
have we explained the term `sub-mapping` anywhere already (documentation).. Can we refer to that.?